packages/veto-app/backend/src/db/index.ts

```diff
@@ -0,0 +1,28 @@
+import Gun from 'gun'
+import { IGunChainReference } from 'gun/types/chain'
+import { Server } from 'http'
+import config from '../utils/config'
+
+let instance: IGunChainReference
+
+const getInstance: Function = (): IGunChainReference => {
```

Comment:
No need to type as `Function`, TS will be able to infer the typing by itself because you properly typed the rest of the function `(): IGunChainReference =>`

Reply:
Removed them